tapeagents/finetune/finetune.py

```diff
@@ -87,6 +84,12 @@ def run_finetuning_loop(
 
     args.gradient_accumulation_passes //= num_processes
     samples_per_pass = num_processes * args.train_batch_size
+    if (ds_plugin := accelerator.state.deepspeed_plugin) is not None:
+        logger.info("Manual inform Deepspeed about micro batch size and gradient accumulation")
```

Comment:
@AlexPiche  you are not propagating the mini batch size down below, but you kept the comment above as is. That's not great. I had to propagate micro batch size because DeepSpeed crashed, not sure why it does not for you. But I think it is the right thing to do any way.